plugins/GSdx/GLLoader.cpp

```diff
@@ -482,4 +490,9 @@ namespace GLLoader {
 
 		s_first_load = false;
 	}
+	
+	#define PRINT_ONCE(stream, ...)	do { \
```

Comment:
The goal is to use the macro.
So you can write `PRINT_ONCE(stderr, "The OpenGL renderer .....");` instead. But macro must come first.